exporter/src/main/scala/uk/gov/nationalarchives/consignmentexport/GraphQlApi.scala

```diff
@@ -27,6 +29,14 @@ class GraphQlApi(keycloak: KeycloakUtils,
     val errorString: String = response.errors.map(_.message).mkString("\n")
   }
 
+  def getConsignmentExport(config: Configuration, consignmentId: UUID) = for {
```

Comment:
Minor thing, but I think calling this "consignmentExport" in the context of the export task is a bit confusing because we're calling two things the "export".

consignementMetadata? consignmentDetails?

Reply:
Fair point